lib/services/elasticsearch.js

```diff
@@ -772,56 +769,50 @@ function ElasticSearch (kuzzle, options, config) {
     return this.client.indices.delete({index: deletedIndexes})
       .then(() => ({deleted: deletedIndexes}))
       .catch(error => Promise.reject(this.formatESError(error)));
-  };
+  }
 
   /**
    * List all known indexes
    *
    * @returns {Promise}
    */
-  this.listIndexes = function elasticsearchListIndexes () {
-    var indexes = [];
-
+  listIndexes() {
     return this.client.indices.getMapping()
       .then(result => {
-        indexes = Object.keys(result);
-        indexes = indexes.filter(indexName => {
-          // @todo : manage internal index properly
-          // exclude empty results
-          return indexName !== '';
-        });
+        let indexes = Object.keys(result)
+          .filter(indexName => indexName !== ''); // @todo : manage internal index properly
```

Comment:
There will be merge conflicts here withPR #723 :
https://github.com/kuzzleio/kuzzle/pull/723/files#diff-8927ec75654a199501c9220ef47f3b5cR789